feedback-mailer/feedback-mailer.gs

```diff
@@ -0,0 +1,210 @@
+const SPEECH_FLOW_IDEAL = "Good";
+const GENERAL_FEATURES_EXPLANATION_IDEAL = "Good";
+const TECH_STACK_EXPLANATION_IDEAL = "Good";
+const CONTRIBUTION_EXPLANATION_IDEAL = "Good";
+const IDEAL_SCORE = 7;
+
+
+/**
+ * Template Default texts
+ */
+
+const FEEDBACK_STUDENT_NAME = "Student Name";
+const FEEDBACK_TEXT = "Feedback Text";
+const FINAL_TEXT = "Final Text";
+
+const MESSAGE_TEMPLATE = `
+Dear ${FEEDBACK_STUDENT_NAME},
+
+We have reviewed your Project Explanation Video and below is the Feedback:
+
+${FEEDBACK_TEXT}
+
+${FINAL_TEXT}
+
+Thanks & Regards,
+The 10x Academy`;
+
+const LINE_BREAK = ".\n";
+const PROJECT_MISMATCH_PRE_TEXT = "Please discuss about the allotted project while explaining. Your allotted project is ";
+const PROJECT_MISMATCH_POST_TEXT = ", but you have discussed about ";
+const VIDEO_STATUS_NON_IDEAL = ". You have not switched on your video, Please switch on your video while project explanation";
+const VIDEO_DURATION_NON_IDEAL_PRE_TEXT = ". Your video duration is ";
+const VIDEO_DURATION_NON_IDEAL_MID_TEXT = ", The ideal video duration is ";
+const PROJECT_EXPLANATION_SCRIPT_NON_IDEAL = ". It feels like you are reading from somewhere (like screen, paper etc.). Practice memorizing your script so, it feels more authentic";
+const SPEECH_FLOW_NON_IDEAL = ". We have observed that your are getting stuck often with your flow of speech. Take necessary pauses to have a free flow of speech";
+const GENERAL_FEATURES_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on Project General Features: ";
+const GENERAL_FEATURES_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 2 - 3 Lines. Please improve on this";
+const TECH_STACK_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on Project Tech Stack: ";
+const TECH_STACK_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 2 - 4 Lines. Please improve on this";
+const CONTRIBUTION_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on your contribution in the Project: ";
+const CONTRIBUTION_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 3 - 6 Lines. Please improve on this";
+
+const FULL_SCORE_TEXT_MAIN = `
+In your project explanation video, you have discussed:
+
+1. Project General Features,
+2. Tech Stack &
+3. Your contribution towards the Project
+
+With a good flow of speech.`;
+
+const FULL_SCORE_TEXT_END = "Thanks for Submitting!"
+const PARTIAL_SCORE_TEXT = "Please create the video again, improving the point(s) discussed above and re-submit the updated video.";
+
+const MAIL_SUBJECT = "Project Explanation Video - Feedback";
+const OWNER_NAME = "The 10x Academy";
+
+
+function afterFormSubmit(e) {
+  const info = e.namedValues;
+  const { body , score } = createMailBody(info);
+  const entryRow = e.range.getRow();
+  const workSheet = SpreadsheetApp.getActiveSheet();
+  workSheet.getRange(entryRow, SCORE_INDEX + 1).setValue(score);
+  workSheet.getRange(entryRow, MAIL_BODY_INDEX + 1).setValue(body);
+  sendEmail(e.namedValues[STUDENT_EMAIL][0], body);
```

Comment:
Based on the response from `sendEmail`, you should also add a column to denote `SENT` or `FAILED`.

Reply:
Added